elife/postgresql-appdb.sls

```diff
@@ -33,6 +33,8 @@ db-perms-to-rds_superuser:
             db_name: {{ db_name }}
             app_user_name: {{ app_user_name }}
             app_user_pass: {{ app_user_pass }}
+        - require:
+            - postgresql-ready # builder-base-formula.postgresql
```

Comment:
isn't that `elife.postgresql` in terms of Salt state files names?

Reply:
sure. it started off life as "builder-base-formula/elife/postgresql.sls" but I shortened it and figured the intent would be obvious. I'll change it

Reply:
fixed